modules/core/sfx.js

```diff
@@ -145,11 +145,24 @@ yargs(hideBin(process.argv))
         })
       } catch (e) {
         consola.error(e)
-
         process.exit(1)
       }
     },
   })
+  .command({
+    command: 'make',
+    description: 'Create new module',
+    handler: async (argv) => {
+      try {
+        const { default: Make } = await import('./src/Make.js')
+        const make = new Make(argv)
+        await make.createModule()
```

Comment:
Make will definitely do other things in the future. Not just create a module. Add some one more argument to the make command so we know we are making a module.

`yarn sfx make module`